docs/guides/quarkus.md

```diff
@@ -0,0 +1,326 @@
+# Quarkus Developer's Guide
+
+When creating a Quarkus function using the `func` CLI, the project directory
+looks like a typical `maven` project. Both HTTP and Event functions have the same
+template structure.
+
+```
+❯ func create my-fn --runtime quarkus 
```

Comment:
I think most or all of the other guides now use `fn` for the name. In any case, having a name with a dash in it in docs is probably not good since the service name would end up being `my--fn`, which is maybe not ideal.